core/src/scene/dataLayer.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "scene/sceneLayer.h"
+#include <string>
+
+namespace Tangram {
+
+// DataLayer represents a top-level layer in the stylesheet, distinct from
+// SceneLayer by its association with a collection within a DataSource
+class DataLayer {
```

Comment:
Right, it's a little funny but this made more sense to me. If `DataLayer` inherited from `SceneLayer` then the class hierarchy is saying that it would be valid for a `DataLayer` to contain other `DataLayer`s, and that's not the case.